apis/cloudbuild/v1alpha1/workerpool_types.go

```diff
@@ -88,6 +88,8 @@ type CloudBuildWorkerPoolObservedState struct {
 	// +required
 	WorkerConfig  *WorkerConfig       `json:"workerConfig,omitempty"`
 	NetworkConfig *NetworkConfigState `json:"networkConfig,omitempty"`
+
+	ETag *string `json:"eTag,omitempty"`
```

Comment:
It's still alpha, so not a blocker but ... I don't know how we should capitalize etag.  I think it might just be `etag`.  I don't think I've seen `eTag` anywhere else (?)

Reply:
Ah! You are right. It's interesting that the camel case format is `etag` with lower case `e` where the capitalized format is `ETag`.

Updated.